src/openforms/js/lang/en.json

```diff
@@ -339,11 +339,6 @@
     "description": "Form step internal name field help text",
     "originalDefault": "Internal name of the form definition used in this form step"
   },
-  "C6xxed": {
-    "defaultMessage": "Example initial",
-    "description": "Variable table initial value title",
-    "originalDefault": "Example initial"
-  },
```

Comment:
why are these files changed?

Reply:
I am just running `./bin/makemessages`

Reply:
I've run makemessages on `master` branch, if you rebase, these changes should no longer be in the diff.